### ISSUE TITLE
There is no scrolling available for retried tests in allure-framework if you try to access it through jenkins

### ISSUE THREAD

**Issue by DenizWork** [allure-framework/allure2#1167]
Hi there, has anyone noticed that on jenkins with allure reporting, there is no scrolling option for the test logs for retries? This can be accessed, if you have a test running on flake and if it failed couple times. The steps to reproduce are, go to your allure test result on jenkins, click on the test suite that failed, on the right side you will see 3 options -> overview, history, and retries. Click on retries and you will see the flake test results. IF you select a previous test result (ex if you have 4 flake runs select the first one not the last), you will see that the redirected page has no scrolling option so I can not see what failed exactly because I can not scroll.

Just wanna say there is no issue when you run tests on flake locally and you check allure report locally, there is scrolling there. Just not on jenkins.

**Comment by dmitryhusev** [comment]
Yes, faced this issue as well. In my case, it was impossible to scroll down screenshot.

**Comment by mrbaco** [comment]
I think there is the similar issue on https://github.com/allure-framework/allure2/issues/753
